packages/grafana-data/src/valueFormats/categories.ts

```diff
@@ -38,7 +38,7 @@ export const getCategories = (): ValueFormatCategory[] => [
   {
     name: 'Misc',
     formats: [
-      { name: 'none', id: 'none', fn: toFixedUnit('') },
+      { name: 'Automatic', id: 'none', fn: toFixedUnit('') },
```

Comment:
Not sure about this, it's not really automatic.

Reply:
Fair!